test/index.js

```diff
@@ -579,6 +618,29 @@ describe('maybe', function() {
       assert(a.chain(a.of).equals(a));
     });
 
+    it('implements Traversable', function() {
+      var u = S.Nothing();
+      var C = Compose(Identity, S.Maybe);
```

Comment:
You can think of the arguments to `Compose` as the types that are expected to have an `of` method attached which are used to implement `of` for `Compose`, so we want to pass `Identity` and `S.Maybe` here.

```js
_Compose.of = function(x) {
  return _Compose(F.of(G.of(x)));
};
```